src/main/java/com/k3ntako/HTTPServer/Main.java

```diff
@@ -15,6 +16,7 @@ public static void main(String[] args) {
     var routes = new HashMap<String, RouteInterface>();
     routes.put("/simple_get", new SimpleGet());
     routes.put("/simple_get_with_body", new SimpleGetWithBody());
+    routes.put("/redirect", new SimpleRedirect());
```

Comment:
I'm thinking if creating these routes should be happening here, in Main class. 2 concerns here:
1 - imagine if we're dealing with not a learning project, but a real-life application, wouldn't it be to messy to put all the routes in here?
2 - think about it from the single responsibility principle point of view - does this class have too many responsibilities now?
Seems like a good time to extract :)